src/NServiceBus.Persistence.AzureTable/Config/WellKnownConfigurationKeys.cs

```diff
@@ -5,9 +5,9 @@ static class WellKnownConfigurationKeys
         public const string SagaStorageCreateSchema = "AzureSagaStorage.CreateSchema";
         public const string SagaStorageAssumeSecondaryIndicesExist = "AzureSagaStorage.SagaStorageAssumeSecondaryIndicesExist";
         public const string SagaStorageAssumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey = "AzureSagaStorage.SagaStorageAssumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey";
+        public const string SagaStorageMigrationMode = "AzureSagaStorage.EnableMigrationMode";
```

Comment:
Convinced now that SagaStorageCompatibilityModel is a better fit

Reply:
That can be addressed in a different PR